### DISCUSSION TITLE
BUG 视频播放问题

### DISCUSSION THREAD

**Discussion by Yan-nian** [Richasy/Bili.Uwp#920] [问答]
电脑为戴尔g15 5511型号
显卡3060
使用时只能正常播放第一个视频，第二个视频就会黑屏但能听到声音和弹幕
安装好所有解码器且每个都切换试
[applog.2022-03-20.log](https://github.com/Richasy/Bili.Uwp/files/8357573/applog.2022-03-20.log)
过 并无效果

**Comment by Richasy** [comment]
尝试在显卡设置里将哔哩设置为独显运行
